# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/python-installer/hotfix.patch
# Copyright (C) 2024 - 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- python-installer/installer/destinations.py	2026-01-30 00:00:00.000000000 +0200
+++ python-installer/installer/destinations.py	2026-08-12 00:00:00.000000000 +0300
@@ -178,9 +178,9 @@
         - Hashes the written content, to determine the entry in the ``RECORD`` file.
         """
         target_path = self._path_with_destdir(scheme, path)
-        if not self.overwrite_existing and target_path.exists():
-            message = f"File already exists: {target_path!s}"
-            raise FileExistsError(message)
+ #       if not self.overwrite_existing and target_path.exists():
+ #           message = f"File already exists: {target_path!s}"
+ #           raise FileExistsError(message)
 
         parent_folder = target_path.parent
         if not parent_folder.exists():
